web-ui/src/main/resources/catalog/components/utility/partials/draftvalidationwidget.html

```diff
@@ -1,20 +1,21 @@
 <a
+  data-ng-show="record"
   href=""
-  data-ng-href="catalog.search#/{{(draft.draft == 'y')?'metadraf':'metadata'}}/{{metadata.uuid}}"
+  data-ng-href="catalog.search#/{{recordIsDraft ? 'metadraf' : 'metadata'}}/{{record.uuid}}"
   class="see-draft-pill"
 >
   <small>
     <small
-      class="valid-status{{draft.valid}} md-type-{{draft.isTemplate}}"
-      title="{{(('draft') | translate)}}: {{(('validStatus-' + draft.valid) | translate)}}"
-      ng-switch="draft.isTemplate"
+      class="valid-status{{record.valid}} md-type-{{record.isTemplate}}"
+      title="{{(title | translate)}}: {{(('validStatus-' + record.valid) | translate)}}"
+      ng-switch="record.isTemplate"
     >
       <i class="fa fa-fw fa-file" ng-switch-when="y" />
       <i class="fa fa-fw fa-pencil" ng-switch-when="n" />
```

Comment:
```suggestion
      <i class="fa fa-fw" data-ng-class="{'fa-pencil':recordIsDraft, 'fa-file-text-o':!recordIsDraft}" ng-switch-when="n" />
```

we could have a `pencil` when in draft, `document` when in not-draft?